src/typeahead/typeahead.directive.ts

```diff
@@ -140,6 +149,9 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
   /**  if false don't focus the input element the typeahead directive is associated with on selection */
     // @Input() protected typeaheadFocusOnSelect:boolean;
 
+  activeDescendant: string;
+  popupId: string;
```

Comment:
Do you need this property? It looks like you don't use it in the directive, you use only `this._container.popupId `

Reply:
it's right, i delete this property